charts/merlin/templates/mlflow-deployment.yaml

```diff
@@ -97,6 +100,9 @@ spec:
       nodeSelector:
 {{ toYaml .Values.mlflow.nodeSelector | indent 8 }}
 {{- end }}
+{{- if and .Values.mlflow.serviceAccount.create }}
+      serviceAccountName: {{ default "mlflow" .Values.mlflow.serviceAccount.name }}
```

Comment:
@shydefoo, I am adding this serviceAccount for mlflow. Which would be connected to a GSA pair elsewhere. So, with this, do you think the above env var `GOOGLE_APP_CREDS` can be removed?

Reply:
I don't think we should remove it completely, rather leave it as an option if for some reason this has to be deployed in a cluster that doesn't have workload identity enabled, we can still use the service account key to access gcp services.